app/models/group.rb

```diff
@@ -12,10 +12,10 @@
 class Group < ActiveRecord::Base
   validates_presence_of :name, :description
 
-  has_many :group_members, foreign_key: :groupid
+  has_many :group_members, foreign_key: :groupid, dependent: :destroy
   has_many :members, -> { order 'name' }, through: :group_members, source: :user
-  has_many :meetings, -> { order 'created_at DESC' }, foreign_key: :groupid
-  has_many :meetings, foreign_key: :groupid
+  has_many :meetings, -> { order 'meetings.created_at DESC' },
```

Comment:
Yeah, I am talking about the former. Because meeting_members is dependent on meetings, and hence is included in the deletion cascade? If so, makes sense, but weird nonetheless.